packages/store/addon/-private/system/store/record-data-store-wrapper.ts

```diff
@@ -1,16 +1,27 @@
 import { RecordDataStoreWrapper as IRecordDataStoreWrapper } from '../../ts-interfaces/record-data-store-wrapper';
+import Store from '../store';
+
+type Store = InstanceType<typeof Store>;
 
 export default class RecordDataStoreWrapper implements IRecordDataStoreWrapper {
-  store: any;
+  store: Store;
   _willUpdateManyArrays: boolean;
   _pendingManyArrayUpdates: string[];
 
-  constructor(store) {
+  constructor(store: Store) {
     this.store = store;
     this._willUpdateManyArrays = false;
     this._pendingManyArrayUpdates = [];
   }
 
+  // TODO this exists just to the default recordData can
+  //  check this in debug for relationships
+  //  we can do away with this with a bit of refactoring
+  // of the relationship layer
+  _hasModelFor(modelName: string) {
```

Comment:
Seems like this is addressed by the CustomModelClasses rfc

Reply:
I think we'll leave it for that RFC implementation to make public in the correct way